### ISSUE TITLE
Bug: Symbol Methods cause SyntaxError: Unexpected token '['

### ISSUE THREAD

**Issue by rezonant** [typescript-rtti/typescript-rtti#25]
Found in `phantomdi`:

```
let result : string = '';
let sym = Symbol();

class A {
    constructor(readonly bar = 'abc') { 
    };

    [sym]() {
        return 123;
    }

    foo(faz : number) { 
        result += faz.toString();
        return 'original';
    } 
};

let i = injector([provide(A), alter(A, { beforeFoo() { } }) ]);
let a2 = i.provide(A);
expect(a2[sym]()).to.equal(123);
```

Error:

```
D:\Dev\phantomdi\dist\injector.test.js:1014
        __RΦ.m("rt:f", "M.")(A.prototype.[sym]);
                                         ^

SyntaxError: Unexpected token '['
    at wrapSafe (internal/modules/cjs/loader.js:979:16)
    at Module._compile (internal/modules/cjs/loader.js:1027:27)
    at Module.replacementCompile (D:\Dev\phantomdi\node_modules\append-transform\index.js:60:13)
    at Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Object.<anonymous> (D:\Dev\phantomdi\node_modules\append-transform\index.js:64:4)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at mapper (D:\Dev\phantomdi\node_modules\require-glob\src\require-glob.js:51:12)
```

**Comment by rezonant** [comment]
The fix ensures we properly handle computed property names, so support is there now for more than just symbols

**Comment by rezonant** [comment]
Released in v0.4.14
